packages/formation/sass/modules/_m-promo-banner.scss

```diff
@@ -0,0 +1,54 @@
+.va-promo-banner {
+  z-index: $base-layer + 1;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+
+  background-color: $color-gray-lightest;
+  color: $color-link-default;
+  font-weight: 700;
+
+  &-body {
+    display: table;
+    width: 100%;
+  }
+
+  &-content, &-close {
+    display: table-cell;
+  }
```

Comment:
I used `display: table` because of the responsive behavior - on smaller screens, the close button's cell should span the whole height of the parent with the close button itself centered vertically and horizontally. I explored flexbox for awhile but there didn't seem to be an easy way to do this. 

![image](https://user-images.githubusercontent.com/1915775/59793210-fa68af80-92a3-11e9-9aed-cd62c3ba703f.png)

Reply:
@ncksllvn The VCL banner has that same behavior that with flexbox, I believe